leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java

```diff
@@ -1,99 +1,105 @@
 package com.schibstedspain.leku.geocoder;
 
+import android.annotation.SuppressLint;
+
 import com.google.android.gms.maps.model.LatLng;
-import hu.akarnokd.rxjava.interop.RxJavaInterop;
+
 import io.reactivex.Scheduler;
 import io.reactivex.android.schedulers.AndroidSchedulers;
 import io.reactivex.disposables.CompositeDisposable;
 import io.reactivex.disposables.Disposable;
+
 import java.util.concurrent.TimeUnit;
-import pl.charmas.android.reactivelocation.ReactiveLocationProvider;
+
+import pl.charmas.android.reactivelocation2.ReactiveLocationProvider;
 
 public class GeocoderPresenter {
-  private static final int RETRY_COUNT = 3;
-
-  private GeocoderViewInterface view;
-  private final GeocoderViewInterface nullView = new GeocoderViewInterface.NullView();
-  private CompositeDisposable compositeDisposable = new CompositeDisposable();
-  private final Scheduler scheduler;
-  private ReactiveLocationProvider locationProvider;
-  private GeocoderRepository geocoderRepository;
-
-  public GeocoderPresenter(ReactiveLocationProvider reactiveLocationProvider, GeocoderRepository geocoderRepository) {
-    this(reactiveLocationProvider, geocoderRepository, AndroidSchedulers.mainThread());
-  }
-
-  public GeocoderPresenter(ReactiveLocationProvider reactiveLocationProvider, GeocoderRepository geocoderRepository, Scheduler scheduler) {
-    this.geocoderRepository = geocoderRepository;
-    this.view = nullView;
-    this.scheduler = scheduler;
-    this.locationProvider = reactiveLocationProvider;
-  }
-
-  public void setUI(GeocoderViewInterface geocoderViewInterface) {
-    this.view = geocoderViewInterface;
-  }
-
-  public void stop() {
-    this.view = nullView;
-    if (compositeDisposable != null) {
-      compositeDisposable.clear();
+    private static final int RETRY_COUNT = 3;
+
+    private GeocoderViewInterface view;
```

Comment:
Why you changed the line space of the whole class? It's confusing for reviewing it.

Reply:
Just changed tabs to spaces. U can just change my previous commit If some code style is wrong in your opinion

Reply:
But that makes checkstyle fail. You would need to revert that change and only modify the lines needed.